R/workhorse.R

```diff
@@ -1,78 +1,145 @@
-build_request <- function(endpoint = NULL,
+build_request <- function(path = NULL,
```

Comment:
Is `path` still needed? Is it here in case someone want to use this for an endpoint you have somehow not put into your internal data about the API?